modules/microshift-low-latency-prepare-workload.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-low-latency-prepare-workload_{context}"]
+= Preparing a {microshift-short} workload for low latency
+
+To take advantage of low latency, workloads running on {microshift-short} must have the `microshift-low-latency` container runtime configuration set using the `RuntimeClass` feature. The CRI-O `RuntimeClass` object is installed with the `microshift-low-latency` RPM, so only the pod annotations need to be configured.
+
+.Prerequisites
+* You installed the `microshift-low-latency` RPM package.
+
+.Procedure
+
+. Use the following example to set the following annotations in the pod spec:
++
+[source,text]
+----
+cpu-load-balancing.crio.io: "disable"
+irq-load-balancing.crio.io: "disable"
+cpu-quota.crio.io: "disable"
+----
++
+.Example pod that runs `oslat` test:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: oslat
+  annotations:
+    cpu-load-balancing.crio.io: "disable" <1>
+    irq-load-balancing.crio.io: "disable" <2>
+    cpu-quota.crio.io: "disable" <3>
+spec:
+  runtimeClassName: microshift-low-latency <4>
+  containers:
+  - name: oslat
+    image: quay.io/container-perf-tools/oslat
+    imagePullPolicy: Always
+    resources:
+      requests:
+        memory: "400Mi"
+        cpu: "10"
+      limits:
+        memory: "400Mi"
+        cpu: "10"
```

Comment:
Because we use `2,4-5` in the blueprint example, let's change it to cpu: 2

```suggestion
        memory: "400Mi"
        cpu: "2"
      limits:
        memory: "400Mi"
        cpu: "2"
```